src/features/user/ManageProjects/components/ProjectSites.tsx

```diff
@@ -487,9 +496,9 @@ function EditSite({ openModal, handleModalClose, changeSiteDetails, siteDetails,
     setGeoJson,
     geoJsonError,
     setGeoJsonError,
-    geoLocation:{
-      geoLatitude:36.96,
-      geoLongitude:-28.5
+    geoLocation: {
+      geoLatitude: 36.96,
```

Comment:
If you search for `36.96` you find a lot of these default values in our code. What is this place?

Reply:
That's the default map center I believe, which we use when we aren't looking at anything specific.

Reply:
Ok, but where does these coordinates come from - is it Atlantis? *just joking* 😁